### DISCUSSION TITLE
Let your drawing move! I made motion-excalidraw, isn't cool?

### DISCUSSION THREAD

**Discussion by ylw5** [excalidraw/excalidraw#6732] [Show and tell]
When I draw a series of interconnected things, I wonder if I can make them move to better express my ideas. So I made a demo, it looks really cool!
I will refine it and it will be a perfect tool for people to show their idea with excalidraw. 
![motion-excali](https://github.com/excalidraw/excalidraw/assets/93325995/e4995b70-c359-49a7-816e-301f6f610309)

**Comment by ylw5** [comment]
I think it would be very useful to have a built-in feature to generate GIFs, as it would help create animated illustrations with excalidraw.

**Comment by jordan-clements** [comment]
Very cool, I like it.
